### DISCUSSION TITLE
Adding My Own Animated Wallpaper

### DISCUSSION THREAD

**Discussion by AdrianeNS** [rocksdanister/lively#2229] [Q&A]
Is there a way to do this?  Do I just need to find a clip of something I want to use and add it to Lively Wallpaper?  I naively misunderstood the addition of AI to the app and thought it would animate an image for me but alas, it did not.  I'd love to have an animated image of a beach with clouds and waves.  Please let me know if there is a way to do this.  thanks!

**Comment by rocksdanister** [comment]
There are AI models that can create animation but they are large in size and require good computational power to run on device.. the other option being using an third party api but then will have to charge users for it due to cost.

You can add your own downloaded video file (mp4, webm..) as wallpaper from stock footage sites likes:
https://www.pexels.com/
https://pixabay.com/
